pygments/lexers/_stan_builtins.py

```diff
@@ -436,122 +601,51 @@
     'multi_normal_prec',
     'multi_student_t',
     'multinomial',
+    'multinomial_logit',
     'neg_binomial',
     'neg_binomial_2',
     'neg_binomial_2_log',
+    'neg_binomial_2_log_glm',
     'normal',
+    'normal_id_glm',
     'ordered_logistic',
+    'ordered_logistic_glm',
+    'ordered_probit',
     'pareto',
     'pareto_type_2',
     'poisson',
     'poisson_log',
+    'poisson_log_glm',
     'rayleigh',
     'scaled_inv_chi_square',
+    'skew_double_exponential',
     'skew_normal',
+    'std_normal',
     'student_t',
     'uniform',
     'von_mises',
     'weibull',
     'wiener',
-    'wishart',
+    'wishart'
 )
 
 RESERVED = (
-    'alignas',
-    'alignof',
-    'and',
-    'and_eq',
-    'asm',
-    'auto',
-    'bitand',
-    'bitor',
-    'bool',
-    'break',
-    'case',
-    'catch',
-    'char',
-    'char16_t',
-    'char32_t',
-    'class',
-    'compl',
-    'const',
-    'const_cast',
-    'constexpr',
-    'continue',
-    'decltype',
-    'default',
-    'delete',
-    'do',
-    'double',
-    'dynamic_cast',
-    'else',
-    'enum',
-    'explicit',
-    'export',
-    'extern',
-    'false',
-    'float',
     'for',
-    'friend',
-    'fvar',
-    'goto',
-    'if',
     'in',
-    'inline',
-    'int',
-    'long',
-    'lp__',
-    'mutable',
-    'namespace',
-    'new',
-    'noexcept',
-    'not',
-    'not_eq',
-    'nullptr',
-    'operator',
-    'or',
-    'or_eq',
-    'private',
-    'protected',
-    'public',
-    'register',
-    'reinterpret_cast',
+    'while',
     'repeat',
-    'return',
-    'short',
-    'signed',
-    'sizeof',
-    'STAN_MAJOR',
-    'STAN_MATH_MAJOR',
-    'STAN_MATH_MINOR',
-    'STAN_MATH_PATCH',
-    'STAN_MINOR',
-    'STAN_PATCH',
-    'static',
-    'static_assert',
-    'static_cast',
-    'struct',
-    'switch',
-    'template',
+    'until',
+    'if',
```

Comment:
Seems duplicated with the keyword list?

Reply:
There are duplicate entries; they were there previously as well. Generally speaking it is true that “if” is a reserved word in Stan, but I don’t know if it needs to be listed in both places in Pygments

Reply:
In any case, they will be highlighted one way or the other. Looking at the lexer, it seems to prefer Keyword over Keyword.Reserved. It's not super important, but it would be nice to deduplicate the list, to make this less confusing.